frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js

```diff
@@ -145,23 +146,40 @@ export async function requestFromStore(
     createRequestBucketInfo(zoomedAddress, resolutions, fourBit, applyAgglomerates, version),
   );
 
-  return doWithToken(async token => {
-    const { buffer: responseBuffer, headers } = await Request.sendJSONReceiveArraybufferWithHeaders(
-      `${dataUrl}/data?token=${token}`,
-      {
-        data: bucketInfo,
-        timeout: REQUEST_TIMEOUT,
-      },
-    );
-    const missingBuckets = parseAsMaybe(headers["missing-buckets"]).getOrElse([]);
-
-    let resultBuffer = responseBuffer;
-    if (fourBit) {
-      resultBuffer = await decodeFourBit(resultBuffer);
-    }
-
-    return sliceBufferIntoPieces(layerInfo, batch, missingBuckets, new Uint8Array(resultBuffer));
-  });
+  try {
+    return doWithToken(async token => {
+      const stuff = await Request.sendJSONReceiveArraybufferWithHeaders(
+        `${dataUrl}/data?token=${token}`,
+        {
+          data: bucketInfo,
+          timeout: REQUEST_TIMEOUT,
+          showErrorToast: false,
+          doNotInvestigate: true,
+        },
+      );
+      const { buffer: responseBuffer, headers } = stuff;
+      const missingBuckets = parseAsMaybe(headers["missing-buckets"]).getOrElse([]);
+
+      let resultBuffer = responseBuffer;
+      if (fourBit) {
+        resultBuffer = await decodeFourBit(resultBuffer);
+      }
+
+      return sliceBufferIntoPieces(layerInfo, batch, missingBuckets, new Uint8Array(resultBuffer));
+    });
+  } catch (errorResponse) {
+    const errorMessage = `Requesting buckets from layer "${
+      layerInfo.name
+    }" failed with status code ${errorResponse.status} - "${errorResponse.statusText}".`;
+    const urlAsString = `URL - ${dataUrl}`;
+    console.error(`${errorMessage} ${urlAsString}`);
+    console.error(errorResponse);
+    Toast.renderDetailedErrorMessage(errorMessage, urlAsString, {
+      sticky: true,
```

Comment:
`sticky: true` means that the toast cannot be closed, right? Maybe that's a bit too aggressive, since there was no error message at all before? Just wondering if we go from one extreme to the other with this change..

Reply:
Also this seems to contradict the PR description which states:

>  (The toast is not sticky, as this error could only be a temporary thing)

Or am I missing something? If it's not sticky, then this code probably needs some throttling (I'd suggest one minute; the throttling key could be the same as the toast-key). Otherwise, the user could close the toast and the toast would immediately re-appear as soon as the next bucket request fails.

Reply:
> Or am I missing something?

Nope, thanks for finding this. Somehow I messed up here as I thought I removed it and overlooked it during the last check ...